installers/Launcher.hs

```diff
@@ -30,19 +29,26 @@ launcherArgs launcher = unwords $
   ]
     where
       nodeArgs = [
+        "--report-server", "http://report-server.aws.iohk.io:8080",
         "--listen", "127.0.0.1:12100",
-        "--report-server", "http://52.59.7.118:8080",
         "--log-config", "log-config-prod.yaml",
         "--update-latest-path", quote (installerPath launcher),
         "--keyfile", quote (runtimePath launcher <> "Secrets" <> (pathSeparator : "secret.key")),
         "--logs-prefix", quote (runtimePath launcher <> "Logs"),
-        "--db-path", quote (runtimePath launcher <> "DB-0.4"),
-        "--wallet-db-path", quote (runtimePath launcher <> "Wallet-0.4"),
-        "--peers-file", "ip-dht-mappings",
-        "--system-start", "1492428658",
+        "--db-path", quote (runtimePath launcher <> "DB-0.5"),
```

Comment:
@gromakovsky can you take a look at these node params. At a first sight they look okish to me. 

I mean - we will know when installers are build are they working as they are suppose too. Master was changed a lot recently so maybe you know some flags are removed/added ?

tls flags are added by me - but I am not sure for the rest of the flags. ystem-start will probably be updated later